ranger/core/actions.py

```diff
@@ -1225,8 +1224,13 @@ def write(string):
             write("%30s = %s\n" % (setting, getattr(self.settings, setting)))
 
         temporary_file.flush()
+        self._call_pager(temporary_file)
+
+    def _call_pager(self, file_):
         pager = os.environ.get('PAGER', ranger.DEFAULT_PAGER)
-        self.run([pager, temporary_file.name])
+        # PAGER might contain arguments, like "less -R"
+        pager = shlex.split(" ")
+        self.run(pager + [file_.name])
```

Comment:
+1, looks good to me in three lines. I'd be somewhat partial towards using a new variable name after the split but do what you think is right.